src/auth/index.js

```diff
@@ -274,4 +274,65 @@ export default class Auth {
     this.storage.remove(authCommon.IMPERSONATION_USER_KEY);
     this.storage.remove(authCommon.IMPERSONATION_REAL_USER_AUTH_KEY);
   }
+
+  parseRedirectFragment(fragment, ourState) {
```

Comment:
I don't love this here, but we are breaking dev and therefore breaking any cloud testing on dev (Harry noticed this)